frontend/src/pages/store/[store].svelte

```diff
@@ -514,8 +515,28 @@
 								maximumFractionDigits: 2,
 							})}
 						</p>
+						{#if item.quantity != null}
+							<p class="text-2xl font-bold">
+								{#if item.quantity == 0}
+									<!-- This is not possible now but it is here if we need it -->
+									<span class="text-red-400"
+										>OUT OF STOCK</span
+									>
+								{:else if item.quantity <= LOW_STOCK}
+									<span class="text-yellow-400"
+										>LOW STOCK</span
+									>
+								{:else}
+									<span class="text-green-400">IN STOCK</span>
+								{/if}
+								<span class="text-gray-400">&nbsp;-&nbsp;</span>
+								<span class="italic"
+									>{item.quantity.toLocaleString()}</span
+								>
```

Comment:
```suggestion
```

Reply:
What was this supposed to do?
The _else_ doesn't fit into the existing _if_ statement because the one above your suggestion already had an _else_

Reply:
Oops. That was supposed to replace the else for the "In Stock" part.

Reply:
You made another [suggestion for the "In Stock" part](https://github.com/chromezoneeths/kitcoin/pull/20#discussion_r783458312), should I use that one?
The only difference is that the other one still says "In Stock"

Reply:
Whole thing:

If out of stock: Out of stock
Else if low: Only {QTY} left!
Else: {QTY} left

Reply:
Could I make the "in stock" part say
> In stock - {QTY} left

I feel like just saying "{QTY} left" is a lot shorter than the others and it looks a bit odd leading with a number to me at least

Also, if you're fine with that, could I keep the quantities as the existing text color and only change the color of the words?
It doesn't matter that much though

Reply:
> Could I make the "in stock" part say
>
> > In stock - {QTY} left
Fine with me

> Also, if you're fine with that, could I keep the quantities as the existing text color and only change the color of the words?

Don't use gray, just don't give it a color class. If you do that, I'm fine with only having the words be colored.

Reply:
![image](https://user-images.githubusercontent.com/37093293/149242561-f9944c11-b5eb-44a1-bbf9-acc25df1c330.png)
![image](https://user-images.githubusercontent.com/37093293/149242633-3f1fbf52-fb99-4223-b7df-7c9585890801.png)

For the colors, I think the "In stock" part looks good but the "Low stock" part looks a bit odd with the color changed right in the middle. 
I also made "left" not colored in for the "In stock" part

Could I change the "Low stock" part to be similar to the other, saying
> Low stock - {QTY} left

If not, I'll just not change the colors for either so they look the same

Reply:
I think just make the whole thing yellow

Reply:
same with the "In stock" part?

Reply:
I think that one looks fine

Reply:
You should bold the numbers though (or maybe underline idk)